src/SchedulingServiceProvider.php

```diff
@@ -27,7 +25,6 @@ public function configureTipoffPackage(TipoffPackage $package): void
                 Game::class => GamePolicy::class,
                 RecurringSchedule::class => RecurringSchedulePolicy::class,
                 ScheduleEraser::class => ScheduleEraserPolicy::class,
-                Slot::class => SlotPolicy::class,
```

Comment:
@drewroberts I'm not pretty sure of it, I think the Slot class would be added it into the BookingsServiceProvider instead

Reply:
Yes, that is correct.